src/main/java/com/google/security/zynamics/reil/interpreter/ReilInterpreter.java

```diff
@@ -73,16 +73,19 @@ public ReilInterpreter(final Endianness endianness, final ICpuPolicy cpuPolicy,
     this.memory = new ReilMemory(endianness);
   }
 
-  private static long getTruncateMask(final OperandSize targetSize) {
+  private static BigInteger getTruncateMask(final OperandSize targetSize) {
     switch (targetSize) {
       case BYTE:
-        return 0xFFL;
+        return BigInteger.valueOf(0xFFL);
       case DWORD:
-        return 0xFFFFFFFFL;
+        return BigInteger.valueOf(0xFFFFFFFFL);
       case QWORD:
-        return 0xFFFFFFFFFFFFFFFFL;
+        return BigInteger.valueOf(0xFFFFFFFFFFFFFFFFL);
+      case OWORD:
+        return BigInteger.valueOf(0xFFFFFFFFFFFFFFL)
+         .shiftLeft(56).add(BigInteger.valueOf(0xFFFFFFFFFFFFFFL)).shiftLeft(16).add(BigInteger.valueOf(0xffffl));
```

Comment:
- Please properly indent
- Stick to a maximum of 100 characters per line
- Consistently use uppercase for hexadecimal constants